### DISCUSSION TITLE
CreateChild with CSLA 6

### DISCUSSION THREAD

**Discussion by JacoJordaan** [MarimerLLC/csla#2843] [Questions] [answered]
Previously you could use the static method on the child class to create a child e.g.:
````
        internal static RequiredPartEdit NewEditableChild()
        {
            return DataPortal.CreateChild<RequiredPartEdit>();
        }
````
How would you do something similar in CSLA 6, especially if you call this from another business object where you are not able to inject the `IChildDataPortalFactory`?

**Comment by michaelcsikos** [comment, marked as answer]
This is basically the same question that I have [here](https://github.com/MarimerLLC/csla/discussions/2838).
